packages/push/src/PushRegistration.ts

```diff
@@ -15,15 +19,85 @@ declare var window: any;
 export class PushRegistration {
 
   public static readonly TYPE: string = "push";
-  public static readonly API_PATH: string = "rest/registry/device";
 
-  private readonly pushConfig?: ServiceConfiguration;
-  private _objectInstance: any;
+  private static readonly API_PATH: string = "rest/registry/device";
+  private static readonly TOKEN_KEY = "UPS_DEVICE_TOKEN";
+
+  private readonly validationError?: string;
+  private readonly variantId?: string;
+  private readonly http?: AxiosInstance;
```

Comment:
I think this should probably be something like `httpClient`

Reply:
Done!